app/routes.py

```diff
@@ -1,2 +1,260 @@
 from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import request, Blueprint, make_response
+from flask import jsonify
+from sqlalchemy import asc, desc
+from datetime import datetime
+import requests
+import os 
+from sqlalchemy import Table, Column, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+ 
 
+
+
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+
+# GET AND CREATE TASK ITEMS
+@tasks_bp.route("", methods=["POST", "GET"], strict_slashes=False)
+def tasks():
+    
+    if request.method == "GET":
+        task_order = request.args.get("sort")
+        if task_order == None:
+            tasks = Task.query.all() # get all items in list
+        elif task_order == "asc": #to sort by ascending 
+            tasks = Task.query.order_by(asc(Task.title))
+        elif task_order == "desc": #to sort by descending 
+            tasks = Task.query.order_by(desc(Task.title))
+
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append(task.display_tasks())
+        return jsonify(tasks_response)
+
+    
+    else:
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return make_response(jsonify({"details": "Invalid data"}), 400)
+            
+        task = Task(title = request_body["title"],
+                            description = request_body["description"],
+                            completed_at = request_body["completed_at"])
+        
+
+        db.session.add(task)
+        db.session.commit()
+
+
+
+        return jsonify({"task": task.display_tasks()}), 201      
+
+# GET, CHANGE, AND DELETE A SPECIFIC TASK ID
+@tasks_bp.route("/<task_id>", methods=["GET", "PUT", "DELETE"], strict_slashes=False)
+def handle_task(task_id):
+
+    task = Task.query.get(task_id)
+
+    if task is None:
```

Comment:
Another way to write these is conditional is:
```suggestion
    if not task:
```